CHANGELOG.md

```diff
@@ -1,78 +0,0 @@
-# Changelog for DP3T-SDK Android
-
-## Version 0.01 (2.6.2020)
-- Frok to get prestandard DP-3T to work like Google/Apple protocoll.
-- Rename to CoraLibre-android-sdk
```

Comment:
These three lines are relevant to coralibre: is it ok to remove them? I mean, changelog files are mostly useless if we will be using GitHub's releases, but anyway I am not completely sure if this should be removed